api/e2e/test/helpers_api_test.go

```diff
@@ -177,6 +177,10 @@ func withDeployedRouter(
 
 	responsePayload, err := io.ReadAll(resp.Body)
 	require.NoError(t, err)
+	t.Logf("Response %s", responsePayload)
+
+	require.True(t, resp.StatusCode >= http.StatusOK && resp.StatusCode <= http.StatusPartialContent,
```

Comment:
are there cases where it return 2XX > 200?

Reply:
It's a bit defensive coding, not really sure whether there is such case for Turing.

Reply:
ok, was just curious if there anything that happen that led to this